.circleci/config.yml

```diff
@@ -681,6 +688,7 @@ workflows:
               ignore: /.*/
       - build_conda_binaries:
           requires:
+            - build_and_test_habitat_lab_conda
```

Comment:
Is this necessary? [CI results of rc2](https://app.circleci.com/pipelines/github/facebookresearch/habitat-sim/4767/workflows/50907ef7-1d28-471a-a9cf-bcaf990e7856) indicate that #1052 fixed this job.

Reply:
Yes, but we would like avoid building the package if the tests aren't passing for Habitat Lab or documentation can't be built.